packages/osx-secure-transport/osx-secure-transport.0.1.1/desr

```diff
@@ -0,0 +1 @@
+macos/ios SecureTransport TLS OSX implementation API for OCaml
```

Comment:
This should be part of the main `opam` file `descr` stanza. The file name is wrong (`desr`->`descr`) can you report the fix upstream?